src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -1203,25 +1205,72 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
                         p += HexsToCharsHexOutput(p, _j, _k);
                         *p++ = '}';
                     }
-                    else
+                    else if ((Ssse3.IsSupported || AdvSimd.IsSupported) && BitConverter.IsLittleEndian)
```

Comment:
```suggestion
                    else if ((Ssse3.IsSupported || AdvSimd.Arm64.IsSupported) && BitConverter.IsLittleEndian)
```
Should it be `Arm64` for the shuffle below?

Reply:
Thanks! Although, I don't think we'll ever add SIMD support for arm32 🙂